tests/explorer.api.tests/TestWebAppFactory.cs

```diff
@@ -101,9 +101,11 @@ public FileInfo GetVcrCasetteInfo(string testClassName, string vcrSessionName)
             return new FileInfo($"../../../.vcr/{testClassName}.{vcrSessionName}.yaml");
         }
 
-        public TimeSpan? GetApiPollingFrequency(FileInfo vcrCassetteInfo)
+        public TimeSpan GetApiPollingFrequency(FileInfo vcrCassetteInfo)
         {
-            return (vcrCassetteInfo.Exists && vcrCassetteInfo.Length > 0) ? TimeSpan.FromMilliseconds(1) : default(TimeSpan?);
+            return (vcrCassetteInfo.Exists && vcrCassetteInfo.Length > 0) ?
```

Comment:
Should maybe check that we are no in VcrMode here - if the file exists and vcr is in record mode, we could end up hammering the api every millisecond...

Reply:
Created issue #66 for this.